charts/artifact-registry/chart/templates/tests/cluster-role-binding.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: test-pods-rb
+subjects:
+  - kind: ServiceAccount
+    name: internal-kubectl
+    namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: test-pods
```

Can we just use the default `cluster-admin` or `view` role and avoid creating our own role?
```suggestion
  name: cluster-admin
```